Makefile

```diff
@@ -193,21 +193,32 @@ TARGETS = \
 	help \
 	info \
 	clean \
-	electron-develop
+	electron-develop \
+	publish-aws
 
 ifeq ($(TARGET_PLATFORM),darwin)
 electron-installer-app-zip: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
 electron-installer-dmg: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).dmg
+publish-aws: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).dmg
```

Comment:
Alternatively (and this helps reducing duplication of the publish targets), can we accumulate the things we want to deploy in a variable, like we do with the targets, and create single deployment targets (e.g: an AWS target) that iterates through them?